src/config.js

```diff
@@ -13,6 +13,11 @@ const config = {
   postgres_port: process.env.POSTGRES_PORT || 5432,
   base_local_url_graphql: `${configGraphQL.local_url}:${configGraphQL.local_port}`,
   base_auth_url_graphql: `${configGraphQL.auth_internal_url}:${configGraphQL.auth_internal_port}`,
+  postgres_backstage_database: process.env.BACKSTAGE_DB_NAME || 'dojot_dash_users',
```

Comment:
remove this
  postgres_user: process.env.POSTGRES_USER || 'postgres',
  postgres_host: process.env.POSTGRES_HOST || 'postgres',
  postgres_database: process.env.POSTGRES_DATABASE || 'dojot_devm',
  postgres_password: process.env.POSTGRES_PASSWORD || 'postgres',
  postgres_port: process.env.POSTGRES_PORT || 5432,